src/stac_utils/google.py

```diff
@@ -262,12 +306,8 @@ def create_table_from_dataframe(
 
     dataframe = dataframe.rename(columns=column_name_conversion)
     table_definition_sql = f"""
-        DROP TABLE IF EXISTS 
-            {project_name}.{dataset_name}.{table_name} 
-        ;
-        CREATE TABLE {project_name}.{dataset_name}.{table_name} ( 
-            {", ".join(column_definitions)}
-        );
+        DROP TABLE IF EXISTS {project_name}.{dataset_name}.{table_name};
```

Comment:
could you make sure the test in test_google.py has the same whitespace as this change, it's causing a test to fail

Reply:
fixed